snowflake/README.md

```diff
@@ -117,6 +117,19 @@ datadog-agent integration install datadog-snowflake==2.0.1
 
 3. [Restart the Agent][6].
 
+#### Collecting Organization Data
+
+By default, this integration monitors the `ACCOUNT_USAGE` schema, but can also monitor the `ORGANIZATION_USAGE` schema to monitor organization-level metrics.
```

Comment:
The usage of _also_ can be misleading. If you set it to collect organization metrics it wont collect account usage metrics, right? Then I think we should say _instead_ instead of _also_. Then mention that you can configure two instances if you want to collect both.